src/ReportWindow.cpp

```diff
@@ -140,7 +139,7 @@ ReportWindow::ReportWindow(BRect frame)
 	prefsLock.Lock();
 	BString reporthelp = gAppPath;
 	prefsLock.Unlock();
-	reporthelp << "helpfiles/" << gCurrentLanguage->Name() << "/Report Window Help";
+	reporthelp << "helpfiles/" << B_TRANSLATE_COMMENT("English","Path to localized helpfiles. Only translate if available in your language.") << "/Report Window Help";
```

Comment:
@pulkomandy is this correct? It's a bit abused to know if the helpfiles are present in the current language.

Reply:
This allows to have languages with translations for the application, but not for the documentation, and fall back to English.

I don't see another way to do that without hardcoding the information in the sourcecode.

Reply:
It could also checks at runtime, but more changes are then needed.